scripts/setup-local-dev.sh

```diff
@@ -11,18 +11,17 @@ export ETHEREUM_WS=http://127.0.0.1:8546;
 
 export ETHEREUM_PRIVATE_KEY="fae42052f82bed612a724fec3632f325f377120592c75bb78adfcceae6470c5a";
```

Comment:
env. variables are no longer needed since deploy is already done

Reply:
Even for augur-node?

Reply:
augur-node doesn't need the env variables, flash uses them